.make/.openshift/fabric8-toggles-config.yml

```diff
@@ -0,0 +1,10 @@
+apiVersion: v1
+items:
+- apiVersion: v1
+  data:
+    fabric8.toggles.url: http://fabric8-toggles-f8toggles.192.168.64.4.xip.io/api
```

Comment:
same problem as reported above: this is tied to the IP address of the minishift VM, which can change (new profile == new IP address, AFAIK)